src/app/profile/profile.component.html

```diff
@@ -0,0 +1,37 @@
+<form [formGroup]="profileForm" (ngSubmit)="onSubmit(profileForm.value)">
+    <h2>Profile</h2>
+    <p>
+        <label for="firstName">First Name</label>
+        <input type="text" id="firstName" formControlName="firstName">
+        <span class="error-message" *ngIf="firstName.hasError('required') && firstName.dirty">Required Field</span>
+    </p>
+    <p>
+        <label for="lastName">Last Name</label>
+        <input type="text" id="lastName" formControlName="lastName">
+        <span class="error-message" *ngIf="lastName.hasError('required') && lastName.dirty">Required Field</span>
+    </p>
+    <p>
+        <label for="email">Email</label>
+        <input type="text" id="email" formControlName="email">
+        <span class="error-message" *ngIf="emailError()">Must include "@" symbol</span>
+    </p>
+    <hr>
+    <h2>Phone Numbers</h2>
+    <div formArrayName="phoneNumbers" *ngFor="let numbers of PhoneNumbersArray.controls; let i = index">
+        <div class="phone-number" [formGroupName]="i">
+            <p>
+                <label for="number">Number {{i}}</label>
+                <input type="text" id="number{{i}}" formControlName="number">
+            </p>
```

Comment:
ISSUE: It was challenging getting the FormGroup to work inside of the FormArray. Specifically when trying to patchValue, it would print just the last number.

Reply:
Not sure I follow, but things appear to be working...